### ISSUE TITLE
feature request: reply

### ISSUE THREAD

**Issue by seth586** [JojiiOfficial/Matrix-EmailBridge#41]
Ideally if I quote reply the body of an incoming email, the bot will send an email reply:

![feature](https://user-images.githubusercontent.com/46291310/108900719-956c6d80-75df-11eb-8f02-62e8fdb78a06.png)

**Comment by JojiiOfficial** [comment]
Good point, although I don't work on this bridge anymore. I'll keep this open since its a handy feature and I still merge PRs.

**Comment by Pacman99** [comment]
Email replies are really simple, they just need the Re: subject line and you just quote the text. So this is something that can already be done if you format your subject right and you quote the original email.

Not to say it can't be better, but this works too.
